package.json

```diff
@@ -49,11 +49,17 @@
       "quantum-circuit": "quantum/nodes/quantum-circuit/quantum-circuit.js",
       "quantum-register": "quantum/nodes/quantum-register/quantum-register.js",
       "classical-register": "quantum/nodes/classical-register/classical-register.js",
+<<<<<<< HEAD
+      "aer-simulator": "quantum/nodes/aer-simulator/aer-simulator.js",
+      "execute": "quantum/nodes/execute/execute.js",
+      "not-gate": "quantum/nodes/not-gate/not-gate.js"
+=======
       "execute": "quantum/nodes/execute/execute.js",
       "measurement": "quantum/nodes/measurement/measurement.js",
       "barrier": "quantum/nodes/barrier/barrier.js",
       "identity": "quantum/nodes/identity/identity.js",
       "hadamard-gate": "quantum/nodes/hadamard-gate/hadamard-gate.js"
+>>>>>>> 2e65462b344e62887ac1efc977cbeb7566944e03
```

Comment:
This code will not compile - the conflict hasn't been dealt with and the conflict markers haven't been removed.